cmd/postgres_exporter/postgres_exporter.go

```diff
@@ -990,11 +993,15 @@ func (s *Servers) GetServer(dsn string) (*Server, error) {
 	var err error
 	var ok bool
 	errCount := 0 // start at zero because we increment before doing work
-	retries := 3
+
+	// Remove this retries and use the global connectionRetries parameter
+	// retries := 3
+
 	var server *Server
 	for {
-		if errCount++; errCount > retries {
-			return nil, err
+		// if errCount++; errCount > retries {
```

Comment:
I know that it is draft, but will be these lines which are commented out deleted after? Or is it for some purpose?

Reply:
It was a draft PR to show where we needed a change. 
Now removed